products/logs/backend/logs_query_runner.py

```diff
@@ -146,7 +146,6 @@ def to_query(self) -> ast.SelectQuery:
         query.where = self.where()
         order_dir = "ASC" if self.query.orderBy == "earliest" else "DESC"
         query.order_by = [
-            parse_order_expr(f"service_name {order_dir}"),
```

Comment:
this shouldn't really be here, it was an attempt at an optimisation but doesn't really make sense (logs don't show up in order), and we'll have service_name filter soon which makes it 100% obsolete